ipmi-config/ipmi-config.h

```diff
@@ -276,6 +276,14 @@ struct ipmi_config_state_data
   int enable_user_after_password_len;
   ipmi_config_enable_user_after_password_t *enable_user_after_password;
 
+  /* lamont: caching to make ipv6 go faster */
+  int ipv6_ipv4_support_initialized;
+  uint8_t ipv6_ipv4_support_channel_number;
+  uint8_t ipv6_ipv4_support_supports_ipv6;
```

Comment:
It turns out that the usage is in formating the parameters in Lan6_Conf.  See ipmi-config/ipmi-config-category-core-lan6-conf-section.c lines 170-225 or so.  The question really becomes whether or not there is benefit in caching the answer, and I suspect it's "no".  I'll plan on axing in its own commit tomorrow morning.